src/_P097_Esp32Touch.ino

```diff
@@ -157,12 +157,17 @@ boolean Plugin_097(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_TEN_PER_SECOND:
     {
-      if (P097_SEND_LONG_PRESS_EVENT &&
-          (p097_touchstart[CONFIG_PIN1] >= 1) &&
-          (timePassedSince(p097_touchstart[CONFIG_PIN1]) >= P097_LONG_PRESS_TIME)) {
-        UserVar.setFloat(event->TaskIndex, 1, 10);
-        eventQueue.add(event->TaskIndex, (getTaskValueName(event->TaskIndex, 1)), 10);
-        p097_touchstart[CONFIG_PIN1] = 0;
+      const int START_PIN = HAS_T0_INPUT ? 0 : 1;
+      const int END_PIN   = LAST_TOUCH_INPUT_INDEX - HAS_T0_INPUT;
+
+      if ((CONFIG_PIN1 >= START_PIN) && (CONFIG_PIN1 <= END_PIN)) {
+        if (P097_SEND_LONG_PRESS_EVENT &&
+            (p097_touchstart[CONFIG_PIN1] >= 1) &&
+            (timePassedSince(p097_touchstart[CONFIG_PIN1]) >= P097_LONG_PRESS_TIME)) {
```

Comment:
What if timePassedSince is way longer, for example a double tap?
In other words, when should the p097_touchStart be cleared without sending a long press event?